pontoon/base/templates/base.html

```diff
@@ -33,9 +33,9 @@
   </head>
 
   <body
-    class="{% block class %}{% endblock %}"
+    class="{% block class %}{% endblock %} {% if request.user.is_authenticated %}{{ user.profile.theme }}{% else %}dark{% endif %}-theme"
```

Comment:
We now repeat this very same line 4 times in total:
```
{% if request.user.is_authenticated %}{{ user.profile.theme }}{% else %}dark{% endif %}
```

It would be nice to factor it out, perhaps into a [custom template tag](https://github.com/mozilla/pontoon/blob/05b851058a0f33d5fc60ef528471d9490898bac0/pontoon/base/templatetags/helpers.py#L30-L36), so we'd just call something like` {{ theme(request.user) }}`.